content/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs.md

```diff
@@ -206,7 +206,7 @@ The `github` context contains information about the workflow run and the event t
 | `github.repository_id` | `string` | {% data reusables.actions.repository_id-description %} |
 | `github.repository_owner` | `string` | The repository owner's username. For example, `octocat`. |
 | `github.repository_owner_id` | `string` | {% data reusables.actions.repository_owner_id-description %} |
-| `github.repositoryUrl` | `string` | The Git URL to the repository. For example, `git://github.com/octocat/hello-world.git`. |
+| `github.repositoryUrl` | `string` | The Git URL to the repository. For example, `git://github.com/octocat/hello-world.git`. [(See note below)](/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#note-on-githubrepositoryurl-property) |
```

Comment:
Suggest that we put the information from your note here directly.

```suggestion
| `github.repositoryUrl` | `string` | **Retired for security reasons.** The URL for the repository with the `git://` protocol, for example, `git://github.com/octocat/hello-world.git`. Replace with {% raw %}`${{ github.serverUrl }}${{ github.repository }}`{% endraw %} to give you the HTTPS URL for the current repository. |
```

Reply:
Thanks, I moved the comment inline with some changes.